### PR TITLE
Add TSG for WINDOWS_CSE_ERROR_CHECK_API_SERVER_CONNECTIVITY error (5)

### PR DESCRIPTION
**Pull request by ShiqianTao**
# Pull request guidance

Thank you for submitting your contribution to our support content! Our team works closely with subject matter experts in CSS and PMs in the product group to review all content requests to ensure technical accuracy and the best customer experience. This process can sometimes take one or more days, so we greatly appreciate your patience.

We also need your help in order to process your request as soon as possible:

- We won't act on your pull request (PR) until you type "**#sign-off**" in a new comment in your pull request (PR) to indicate that your changes are complete.

- After you sign off in your PR, the article will be tech reviewed by the PM or SME if it has more than minor changes. Once the article is approved, it will undergo a final editing pass before being merged.

### CODE REVIEW

**Comment by axelgMS** [comment]
LGTM

File: support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Troubleshooting
+
+Run a Test-NetConnection command to verify that your nodes can resolve the cluster's fully qualified domain name (FQDN):
+
+If you have existing Windows nodes, you can run:
+```powershell
+Test-NetConnection -ComputerName <cluster-fqdn> -Port 443
+```
+
+Or you must have Linux nodes, you can run
```
Comment (przlplx) [inline support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md:39]: "you can run:"

File: support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+```shell
+nc -vz <cluster-fqdn> 443
+```
+
+## Solution
+
+You should check your network configuration if the above result shows "False" or "Timeout". E.g., check whether you have `Deny` rules for API Server in VNet NSG (Network Security Groups).
```
Comment (przlplx) [inline support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md:46]: [or "Timeout." For example, check...]

File: support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+If you're using egress filtering through a firewall, make sure that traffic is allowed to your cluster FQDN.
+
+In rare cases, the firewall's outbound IP address can be blocked if you've authorized IP addresses that are enabled on your cluster. In this scenario, you must add the outbound IP address of your firewall to the list of authorized IP ranges for the cluster. For more information, see [Secure access to the API server using authorized IP address ranges in AKS](/azure/aks/api-server-authorized-ip-ranges).
+
+## More information
+
+- [General troubleshooting of AKS cluster creation issues](troubleshoot-aks-cluster-creation-issues.md)
+
+[!INCLUDE [Azure Help Support](../../includes/azure-help-support.md)]
```
Comment (przlplx) [inline support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md:56]: add the following disclaimer at the end (for the "https://linuxcommandlibrary.com/man/netcat" link):

[!INCLUDE [Third-party disclaimer](../../../includes/third-party-contact-disclaimer.md)]

File: support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md (hunk 1 above)
Comment (ShiqianTao) [reply support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md]: Updated. Thank you.

File: support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md (hunk 2 above)
Comment (ShiqianTao) [reply support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md]: Updated. Thank you.

File: support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md (hunk 3 above)
Comment (ShiqianTao) [reply support/azure/azure-kubernetes/error-code-windows-cse-error-check-api-server-connectivity.md:56]: Updated. Thank you.

**Comment by ShiqianTao** [comment]
#sign-off

**Review by AbelHu — APPROVED** [review]
LGTM. Thanks for the PR.

**Review by junjiezhang1997 — APPROVED** [review]

**Comment by genlin** [comment]
#sign-off

**Comment by AmandaAZ** [comment]
@ShiqianTao Thanks for your contribution. Before merging this PR, I will create a private version of this PR in the private repository to do an editorial review and then send the preview version of this article to you for confirmation.

**Comment by AmandaAZ** [comment]
@ShiqianTao The private version PR https://github.com/MicrosoftDocs/SupportArticles-docs-pr/pull/5147 is merged, so we closed this public PR. Thanks.

**Pull request closed without merging**